internal/view/app.go

```diff
@@ -691,6 +693,54 @@ func (a *App) helpCmd(evt *tcell.EventKey) *tcell.EventKey {
 	return nil
 }
 
+// previousView returns to the view prior to the current one in the history
+func (a *App) previousView(evt *tcell.EventKey) *tcell.EventKey {
+	if evt != nil && evt.Rune() == 'b' && a.Prompt().InCmdMode() {
```

Comment:
IMHO `evt.Rune() == 'b'` should use a variable for the comparison, not a hardcoded value. This would potentially allow rebinding the key in the future.